packages/crypto-wallet-core/src/transactions/doge/index.ts

```diff
@@ -0,0 +1,25 @@
+import { BTCTxProvider } from '../btc';
+
+export class DOGETxProvider extends BTCTxProvider {
+  lib = require('bitcore-lib-doge');
+  create({ recipients, utxos = [], change, wallet, fee = 20000 }) {
+    change = change || wallet.deriveAddress(wallet.addressIndex, true);
+    const filteredUtxos = this.selectCoins(recipients, utxos, fee);
+    const btcUtxos = filteredUtxos.map(utxo => {
+      const btcUtxo = Object.assign({}, utxo, {
+        amount: utxo.value / 1e8,
+        txid: utxo.mintTxid,
+        outputIndex: utxo.mintIndex
+      });
+      return new this.lib.Transaction.UnspentOutput(btcUtxo);
+    });
+    let tx = new this.lib.Transaction().from(btcUtxos).feePerByte(Number(fee) + 2);
```

Comment:
https://github.com/bitpay/bitcore/blob/master/packages/crypto-wallet-core/src/transactions/btc/index.ts

It looks like there's supposed to be a feeRate param for setting feePerByte, fee is supposed to set the total fee.

Does this not work?
```
export class DOGETxProvider extends BTCTxProvider {
  lib = require('bitcore-lib-doge');
}
```